docs/standard_library.md

```diff
@@ -0,0 +1,57 @@
+# Standard library
+
+Deno provides a set of standard modules that are audited by the core team and
+are guaranteed to work with Deno.
+
+Standard library is available at: https://deno.land/std/
+
+## Versioning and stability
+
+Standard library is not yet stable and therefore it is versioned differently
+than Deno. For latest release consult `std/` website.
+
+We strongly suggest to always use imports with pinned version of standard
+library to avoid unintended changes.
+
+## Troubleshooting
+
+Some of the modules provided in standard library use unstable Deno APIs.
+
+Trying to run such modules without `--unstable` CLI flag ends up with a lot of
+TypeScript errors suggesting that some APIs on `Deno` namespace do not exist:
+
+```typescript
+// main.ts
+import { copy } from "https://deno.land/std@v0.50.0/fs/copy.ts";
```

Comment:
Doesn't the currently favoured way exclude the `v` prefix in the branch?

Reply:
Right :) feel free to add more comments, you've answered a lot of questions related to this problem